added train_from_model folder

caliskate opened the pull request:
added train_from_model folder and notebook with the following changes:
moved data.yaml, model, and dataset locations to LOCAL dir preventing model training path errors
updated yaml.data to all 10 construction classes to prevent model training IndexError

added markdowns
added suggested WIP
added new datasets link

added label dir splitting process
prevented raw_images and raw_labels from being cleared to keep for multiple runs

trained model from custom labels and construction dataset labels
added model result images
added print for model train execution time
used model to draw bbox instead of function
added test image and predicted image *from custom dataset*
cleared runs and predictions for each execution

caliskate committed:
added train_from_model folder

added train_from_model folder

victorhg commented:
Look good to me

victorhg reviewed (approved):
Looks good for us to try. Suggestion for later is to incorporate files to the main structure, although I think this is is better for understanding

caliskate committed:
improved model: image augmentations and 60 epochs

change notes 11/24/24:

added image augmentations to custom dataset for model training
updated model training with 60 epochs
updated wip and added 11/24 change notes

caliskate committed:
updated_train_from_custom

change notes updated 11/24/24

added image augmentations to custom dataset for model training
updated model training to 60 epochs from 30 epochs
updated wip and added 11/24 change notes
prediction Intersection Over Union parameter (iou) decreased to 0.4 from 0.7

caliskate committed:
updated train_from_custom notebook

change notes 11/24/24

added image augmentations to custom dataset for model training
updated model training to 60 epochs from 30 epochs
updated wip and added 11/24 change notes

caliskate commented:
change notes 11/24/24

added image augmentations to custom dataset for model training to better predict objects in custom images
updated model training to 60 epochs from 30 epochs, improved overall confidence scores
updated wip and 11/24 change notes

caliskate committed:
updated photo

updated test photo output
